gprofiler/merge.py

```diff
@@ -45,6 +45,7 @@ def parse_one_collapsed(collapsed: str, add_comm: Optional[str] = None) -> Stack
             continue
         try:
             stack, _, count = line.rpartition(" ")
+            stack = stack.replace("|", ";")
```

Comment:
The backend then splits again by `;`. As long as we remain with the basic stackcollapse format, we can only do this translation in the backend.